packages/wallets/src/features/cashier/modules/CashierLocked/CashierLocked.tsx

```diff
@@ -36,7 +36,7 @@ const CashierLocked: React.FC<TCashierLockedProps> = ({ children, module }) => {
     const noResidence = cashierValidation?.no_residence;
 
     const isSystemMaintenance = cashierValidation?.system_maintenance && !isVirtual;
-    const isCashierLocked = status?.is_cashier_locked && !isVirtual;
+    const isCashierLocked = status?.is_cashier_locked && !isVirtual && !isFetchingAccountStatus;
```

Comment:
account_status was being refetched, thus it momentarily shows `status.is_cashier_locked` as `true` when switching from demo to real account, added condition to check if its fetching the data again

https://github.com/binary-com/deriv-app/assets/103016120/21e37e89-555c-477d-8466-a6c275618d90